servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ParentConnectionContext.java

```diff
@@ -162,7 +166,11 @@ protected final void doCloseAsyncGracefully() {
         }, true);
     }
 
-    private void notifyOnClosingImpl() {    // For access from AbstractH2ParentConnection
+    /**
+     * This method is required to access notifyOnClosing() from AbstractH2ParentConnection, because usage of
+     * {@code parentContext.notifyOnClosing()} directly triggers {@link java.lang.IllegalAccessError}.
+     */
+    private void notifyOnClosingImpl() {
```

Comment:
Removal of this method was the cause for `H2PriorKnowledgeFeatureParityTest > serverGracefulClose` failures 🤦 
I'm not sure why Intellij IDEA or javac could not catch this as a problem.

```
    java.lang.BootstrapMethodError: java.lang.IllegalAccessError: tried to access method io.servicetalk.transport.netty.internal.NettyChannelListenableAsyncCloseable.notifyOnClosing()V from class io.servicetalk.http.netty.H2ParentConnectionContext$AbstractH2ParentConnection
    	at io.servicetalk.http.netty.H2ParentConnectionContext$AbstractH2ParentConnection.channelRead(H2ParentConnectionContext.java:291) ~[servicetalk-http-netty-0.42.39-SNAPSHOT.jar:?]
```